### ISSUE TITLE
SC | Fix focus on first error trap

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#98137]
## What happened?

On Supplemental Claim add VA evidence page which has multiple required fields:
- Without adding anything, activate the "Continue" button
- Errors for all fields are shown & the first error message (span) is focused
- When the field is blurred, focus will return to the first error message (span)
- This essentially creates a focus lock on the first field

![focus trap between first error message and input](https://github.com/user-attachments/assets/3389e8e6-c007-45eb-bbfa-b797979dffe9)

## How severe is the problem?

- [x] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [ ] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [ ] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. Focus should only move to the first error upon page submission (using continue)
2. Focus is not altered upon blur of a field

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place
